### DISCUSSION TITLE
Settings

### DISCUSSION THREAD

**Discussion by revk** [revk/xESP32-Faikout#224] [Ideas]
There are now a lot of settings, and I am very pleased as to how techie everyone is here, thank you.

Some settings show on the web interface, but far from all...

So, ideas.

1. I really need to find a way to "lock" down settings, in a sane way, on a local non https interface. Ideas welcome. It may be that a simple password is "good enough" for a simple local control to stop people messing. Yes, eventually "matter", I got some way then had a stroke, OK, so leave that for a bit.
2. I could have an "advanced" setting which lists a lot (maybe all) of the settings on the settings page, so no MQTT needed.

So, thoughts on those ideas?

**Comment by turbolooser** [comment]
If possible I would keep the basic settings as is (but maybe with the older order then the current settings page on beta)
and then add a section called "advanced" to give it all settings with the clear message that this could screw up stuff

- I also would like to see some backup/restore and set to default button in that advanced section if possible
- I would get rid of the list of scanned wifi's in the area.

**Reply by revk** [reply]
Interesting

**Reply by revk** [reply]
The scanned wifi are mainly for initial set up. Maybe the whole "wifi settings" is what needs to go when on line, and be something separate specially for initial set up, or if you need it...

**Reply by turbolooser** [reply]
A "reboot" button would also be nice

**Reply by revk** [reply]
Always an interesting question - why?

**Reply by turbolooser** [reply]
To ensure that the faikin is not having any issue/memory leak/hanging something. 
My question would be "Why not?" (if this is technically easy to do)

**Reply by revk** [reply]
MQTT `command/Faikin/restart`

**Comment by revk** [comment]
Latest beta has a number of changes. Should be issued shortly.

**Reply by turbolooser** [reply]
cool.... that looks really awesome 👍 
so many nice options to play with ;-)

**Reply by revk** [reply]
Yeh, I need to add more explanations on some of them :-)

**Reply by turbolooser** [reply]
yeah thats fine.... maybe a link to the advanced documentation could do the trick rather than updating the firmware all the time.

**Reply by njh** [reply]
I like the new settings page 🙂 
Great to have these easy to toggle, without looking up MQTT topic.

**Comment by njh** [comment]
Page `<title>` is "WiFi Setup" - but it actually covers a lot more than that now.

**Reply by revk** [reply]
Fixed in next beta, thanks.

**Comment by njh** [comment]
I don't personally use "Faikin-auto mode". Does it make sense to have that turned on/off on the settings page - rather than cluttering the home page, for people who don't use it?

**Reply by revk** [reply]
If you use it, then it is the basic temp control for your system, so yes. I could have a setting to remove it.
